### DISCUSSION TITLE
Feedback Sprint 1

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#260] [General]
- Necesitamos un footer que se agregue a todas las páginas, tendría información de copyright y link a la compañía por ahora.
- Crear un PDF con los diagramas o tres PDFs distintos para colocar en Documentación (primero tendríamos que actualizar los gráficos para que reflejen el código que está escrito, eso ya lo había considerado)
- Navegación en modo invitado: veremos que dice sobre la navegación porque salta de modo invitado a modo pujador a modo invitado dependiendo de los clicks que hace.

El resto está bien. Luego abro issues con las dos cosas que pidió acá.

**Comment by NataliaAlvarezIspc** [comment]
Me ocupo del footer, asi que cuando la crees asignamela directamente

**Comment by rpgrca** [comment]
Me di cuenta que ya tenemos en Documentación el archivo con los gráficos y las explicaciones así que le hice un addenum a mi mensaje. Veremos que responde.
